admin/notifiers/class-configuration-notifier.php

```diff
@@ -0,0 +1,175 @@
+<?php
+/**
+ * WPSEO plugin file.
+ *
+ * @package WPSEO\Admin\Notifiers
+ */
+
+/**
+ * Represents the logic for showing the notification.
+ */
+class WPSEO_Configuration_Notifier implements WPSEO_Listener {
+
+	/**
+	 * Option name used to determine whether the notice has been dismissed.
+	 *
+	 * @var string
+	 */
+	const META_NAME = 'wpseo-dismiss-configuration-notice';
+
+	/**
+	 * Default value.
+	 *
+	 * @var string
+	 */
+	const META_VALUE = 'yes';
+
+	/**
+	 * Should the notification be shown.
+	 *
+	 * @var bool
+	 */
+	protected $show_notification;
+
+	/**
+	 * Constructs the object by setting the show notification property based the given options.
+	 */
+	public function __construct() {
+		$this->show_notification = WPSEO_Options::get( 'show_onboarding_notice', false );
+	}
+
+	/**
+	 * Returns the content of the notification.
+	 *
+	 * @return string A string with the notification HTML, or empty string when no notification is needed.
+	 */
+	public function notify() {
+		if ( ! $this->show_notification() ) {
+			return $this->re_run_notification();
+		}
+
+		if ( WPSEO_Options::get( 'started_configuration_wizard' ) ) {
+			return $this->continue_notification();
+		}
+
+		return $this->first_time_notification();
+	}
+
+	/**
+	 * Listens to an argument in the request URL. When triggered just sets the notification to dismissed.
+	 *
+	 * @return void
+	 */
+	public function listen() {
+		if ( ! $this->show_notification() || ! $this->dismissal_is_triggered() ) {
+			return;
+		}
+
+		$this->set_dismissed();
+	}
+
+	/**
+	 * Checks if the dismissal should be triggered.
+	 *
+	 * @return bool True when action has been triggered.
+	 */
+	protected function dismissal_is_triggered() {
+		return filter_input( INPUT_GET, 'dismiss_get_started' ) === '1';
```

Comment:
It was used as a URL parameter for the button to _dismiss_ the notification. The notification is not dismissible now.